port/unix/omrsysinfo.c

```diff
@@ -256,6 +256,18 @@ struct  {
 #if defined(LINUX)
 
 #define OMR_CGROUP_V1_MOUNT_POINT "/sys/fs/cgroup"
+#define ROOT_CGROUP "/"
+
+/* Following is the description of /proc/<pid>/cgroup copied from 'man' page for proc:
```

Comment:
We can't copy from the `man` page for proc as the documentation is under GPLv2: http://man7.org/linux/man-pages/man5/proc.5.license.html

Reply:
I updated the comment to put it in my own words instead of copying from `man` page.